modules/offers/client/views/directives/tr-offer-host-view.client.view.html

```diff
@@ -198,6 +198,11 @@ <h4 ng-if="!trOfferHost.isOwnOffer">
            class="btn btn-sm btn-inverse-primary">
           Bigger map
         </a>
+        <a ng-if="trOfferHost.isMobile"
```

Comment:
Let's try one-time-binding (`::`) here so that `isMobile` wouldn't be evaluated more than once (and not on every scope-render): https://code.angularjs.org/1.5.11/docs/guide/expression#one-time-binding

Reply:
Done